code/screens/case-studies/components/left.js

```diff
@@ -0,0 +1,38 @@
+import Link from "../../../common/btns/link";
+import React from "react";
+import Tags from "../../../common/tag";
+import animations from "../../../../assets/js/animations";
+import { PROJECTS_INFO } from "../../../../assets/js/common/case-studies-projects";
+
+function Left({ type }) {
+  const { title, description, images, button, tags } = PROJECTS_INFO[type];
+  const { url, btnText } = button;
+  const { mockup } = images;
+  return (
+    <div className="case-studies flex-between">
```

Comment:
Can u cut some wrappers? Or combine them into 1 component?

Reply:
I can combine, but only two together

Reply:
Do it